### ISSUE TITLE
Milestone 3

### ISSUE THREAD

**Issue by michielbdejong** [federatedbookkeeping/task-tracking#5]
Copied from [the proposal](https://github.com/federatedbookkeeping/task-tracking/blob/main/ngi-assure-application.md#budget):
Milestone 3 runs in parallel with 2, and updates the federation, using the protocol established in Milestone 1, of the multiple task-tracking systems. It also integrates additional third-party systems, both open- and closed-source, Prejournal, timeld, and Tiki (multiple instances), integrating with additional external systems GitHub, SolidOS, Google Calendar, and Google Sheets as a minimum, with a stretch goal of GitLab, Toggl, and InvoiceNinja if achievable within the budget. This breaks down as follows:

3a Apply updated federation protocol with additional capabilities to Tiki federation with Prejournal, timeld and additional Tiki instance. Estimated effort: 37,5h (VE: 30h, GS: 7,5h)

3b: Apply updated federation protocol with additional capabilities to Prejournal federation with timeld and Tiki (multi-instance). Estimated effort: 22,5h (MdeJ: 15h, GS: 7,5h)

3c: Integrate Federation with suitably popular third-party task-tracking systems (e.g. GitHub). Estimated effort: 60h (MdeJ: 60h)

3d: Integrate other systems as appropriate; complete testing and trouble-shooting. Estimated effort: 30h.

**Comment by michielbdejong** [comment]
Moved to [Milestone 3](https://github.com/federatedbookkeeping/task-tracking/milestone/3).
